src/Microsoft.AspNet.Identity.EntityFramework/IdentityEntityFrameworkServices.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNet.Identity
     /// <summary>
     /// Default services
     /// </summary>
-    public class IdentityEntityFrameworkServices
+    internal class IdentityEntityFrameworkServices
```

Comment:
I don't think we should make this internal. @haok didn't we get rid of this pattern in other repos?